src/midisequencer.cpp

```diff
@@ -246,6 +260,33 @@ namespace midisequencer{
         }
     }
 
+    float sequencer::get_start_skipping_silence() {
+        for (auto i = messages.begin(); i != messages.end(); ++i) {
+            // If we find Loop Start before the first NoteOn, just start there
+            if (is_loop_start(i->message)) {
+                float time = i->time;
+                // RPG_RT always rewinds "a little"
+                // This amount is based on the tempo, and this 2100000 divisor
+                // I determined experimentally.
+                time = std::max(0.0f, time - (i->tempo / 2100000.0f));
```

Comment:
Leaving a note here, maybe we can confirm exact numbers from RE tools? Or maybe it's too hard if it's buried in harmony.dll?

Reply:
I couldn't figure out the harmony code yet but yeah this should be verified in RE tools.

Reply:
I have no experience in doing this myself but if any one of you smart folk work it out, I'd be very keen to hear how close my experimentally-derived value was ;)

Reply:
I create a new issue for this